src/containers/InputContainer.js

```diff
@@ -0,0 +1,98 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+// styled component를 사용한다
+
+const AppInput = styled.div`
+  width: 28rem;
+  margin-bottom: 1rem;
+`;
+
+const FormControl = styled.form`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  width: 100%;
+`;
+
+const FormInput = styled.input`
+  width: 23rem;
+  padding: 0.75rem;
+  font-size: 1rem;
+  border-radius: 2rem;
+  border-color: transparent;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+  padding-left: 1.2rem;
+  font-family: var(--ff-main);
+  &:hover {
+    box-shadow: 0 0 10px rgba(255, 255, 255, 0.6);
+  }
+`;
+
+const AppTitle = styled.div`
+  font-size: 1.7rem;
+  display: flex;
+  text-align: center;
+  justify-content: center;
+  margin: 1.5rem 0 1.5rem 0;
```

Comment:
아래와 같이 작성해도 똑같이 작동합니다!
```suggestion
  margin: 1.5rem 0;
```

Reply:
아 이런 식으로 되는 군요. 몰랐습니다, 코멘트 감사드려요!